talk/tools.tex

```diff
@@ -662,8 +662,8 @@
       proprietary tool, the most complete
     \item[\href{http://cppcheck.sourceforge.net/}{\beamergotobutton{cppcheck}}]
       free and opensource, but less complete
-    \item[\href{http://clang-analyzer.llvm.org/}{\beamergotobutton{scan-build}}]
-      the clang source analyzer
+    \item[\href{https://clang.llvm.org/extra/clang-tidy/}{\beamergotobutton{clang-tidy}}]
+      clang-based ``linter'', includes clang static analyzer
```

Comment:
clang-tidy fully absorbs the clang static analyzer, so I felt we can replace it here.

Reply:
Absolutely